docs: added TIM references to markdown (#322)
terraform-ibm-modules/stack-ibm-core-security-services: reference-architectures/reference-architecture-css.md

1. {{site.data.keyword.keymanagementserviceshort}}

>   {{site.data.keyword.keymanagementserviceshort}} is responsible for centrally managing the lifecycle of encryption keys that are used by {{site.data.keyword.cos_full_notm}} buckets, {{site.data.keyword.secrets-manager_short}}, and event notification resources. Additionally, it can manage encryption keys for any customer workload that requires protection. To automate provisioning of {{site.data.keyword.keymanagementserviceshort}} with key rings and keys as code, you can use [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) for [{{site.data.keyword.keymanagementserviceshort}}](https://registry.terraform.io/modules/terraform-ibm-modules/kms-all-inclusive/ibm/latest){: external}.

Reviewer: ```suggestion
  {{site.data.keyword.keymanagementserviceshort}} is responsible for centrally managing the lifecycle of encryption keys that are used by {{site.data.keyword.cos_full_notm}} buckets, {{site.data.keyword.secrets-manager_short}}, and event notification resources. Additionally, it can manage encryption keys for any customer workload that requires protection. To automate provisioning of {{site.data.keyword.keymanagementserviceshort}} with Key rings and Keys , you can use [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) for [{{site.data.keyword.keymanagementserviceshort}}](https://registry.terraform.io/modules/terraform-ibm-modules/kms-all-inclusive/ibm/latest) {: external}.
```

  ```terraform
  module "kms" {
    source  = "terraform-ibm-modules/kms-all-inclusive/ibm"
    version = "<version>"

    resource_group_id         = "<resource_group_id>"
    region                    = "<region>"
    key_protect_instance_name = "<instance_name>"
    keys = [
      {
        key_ring_name = "<key_ring_name>"
        keys = [
          {
            key_name     = "<root_key_name>"
            force_delete = true
          }
        ]
      }
    ]
  }
  ```
  {: codeblock}

2. {{site.data.keyword.secrets-manager_short}}

>   {{site.data.keyword.secrets-manager_short}} securely stores and manages sensitive information, including API keys, credentials, and certificates. It uses encryption keys from {{site.data.keyword.keymanagementserviceshort}} to encrypt sensitive data and to seal and unseal vaults that hold the secrets. It is preconfigured to send events to the {{site.data.keyword.en_short}} service, allowing customers to set up email or SMS notifications. Moreover, it is automatically configured to forward all API logs to the customer's logging instance. To automate provisioning as code, you can use [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) for [{{site.data.keyword.secrets-manager_short}}](https://registry.terraform.io/modules/terraform-ibm-modules/secrets-manager/ibm/latest){: external}.

Reviewer: ```suggestion
  {{site.data.keyword.secrets-manager_short}} securely stores and manages sensitive information, including API keys, credentials, and certificates. It uses encryption keys from {{site.data.keyword.keymanagementserviceshort}} to encrypt sensitive data and to seal and unseal vaults that hold the secrets. It is preconfigured to send events to the {{site.data.keyword.en_short}} service, allowing customers to set up email or SMS notifications. Moreover, it is automatically configured to forward all API logs to the customer's logging instance. To automate provisioning , you can use [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) for [{{site.data.keyword.secrets-manager_short}}](https://registry.terraform.io/modules/terraform-ibm-modules/secrets-manager/ibm/latest) {: external}.
```

  ```terraform
  module "secrets_manager" {
    source  = "terraform-ibm-modules/secrets-manager/ibm"
    version = "<version>"

    resource_group_id    = "<resource_group_id>"
    region               = "<region>"
    secrets_manager_name = "<instance_name>"
  }
  ```
  {: codeblock}

3. {{site.data.keyword.sysdigsecure_full_notm}}

>   The {{site.data.keyword.sysdigsecure_full_notm}} instance is pre-configured with Cloud Security Posture Management (CSPM) enabled using the Configuration Aggregator features from the App Configuration instance that is also provisioned as part of this solution. To automate provisioning, you can use [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) for [{{site.data.keyword.keymanagementserviceshort}}](https://registry.terraform.io/modules/terraform-ibm-modules/kms-all-inclusive/ibm/latest){: external}.

Reviewer: How is referencing KMS all inclusive relevant here as the point is about sysdig? This line looks a bit odd - 
```
you can use [Terraform IBM Modules (TIM)](...) for [{{site.data.keyword.keymanagementserviceshort}}](https://registry.terraform.io/modules/terraform-ibm-modules/kms-all-inclusive/ibm/latest)
```

>   ```terraform

Reviewer: Please provide some spacing between sample code and textual information.

1. {{site.data.keyword.keymanagementserviceshort}}

>   {{site.data.keyword.keymanagementserviceshort}} is responsible for centrally managing the lifecycle of encryption keys that are used by {{site.data.keyword.cos_full_notm}} buckets, {{site.data.keyword.secrets-manager_short}}, and event notification resources. Additionally, it can manage encryption keys for any customer workload that requires protection. To automate provisioning of {{site.data.keyword.keymanagementserviceshort}} with key rings and keys as code, you can use the [KMS All-Inclusive module](https://registry.terraform.io/modules/terraform-ibm-modules/kms-all-inclusive/ibm/latest){: external} from [Terraform IBM Modules](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim).

Reviewer 2: ```suggestion
  {{site.data.keyword.keymanagementserviceshort}} is responsible for centrally managing the lifecycle of encryption keys that are used by {{site.data.keyword.cos_full_notm}} buckets, {{site.data.keyword.secrets-manager_short}}, and event notification resources. Additionally, it can manage encryption keys for any customer workload that requires protection. To automate provisioning of {{site.data.keyword.keymanagementserviceshort}} with key rings and keys as code, you can use [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) for [{{site.data.keyword.keymanagementserviceshort}}](https://registry.terraform.io/modules/terraform-ibm-modules/kms-all-inclusive/ibm/latest){: external}.
```

  ```terraform
  module "kms" {
    source  = "terraform-ibm-modules/kms-all-inclusive/ibm"
    version = "<version>"

    resource_group_id         = "<resource_group_id>"
    region                    = "<region>"
    key_protect_instance_name = "<instance_name>"
    keys = [
      {
        key_ring_name = "<key_ring_name>"
        keys = [
          {
            key_name     = "<root_key_name>"
            force_delete = true
          }
        ]
      }
    ]
  }
  ```
  {: codeblock}

2. {{site.data.keyword.secrets-manager_short}}

>   {{site.data.keyword.secrets-manager_short}} securely stores and manages sensitive information, including API keys, credentials, and certificates. It uses encryption keys from {{site.data.keyword.keymanagementserviceshort}} to encrypt sensitive data and to seal and unseal vaults that hold the secrets. It is preconfigured to send events to the {{site.data.keyword.en_short}} service, allowing customers to set up email or SMS notifications. Moreover, it is automatically configured to forward all API logs to the customer's logging instance. To automate provisioning as code, you can use the [Secrets Manager module](https://registry.terraform.io/modules/terraform-ibm-modules/secrets-manager/ibm/latest){: external} from [Terraform IBM Modules](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim).

Reviewer 2: ```suggestion
  {{site.data.keyword.secrets-manager_short}} securely stores and manages sensitive information, including API keys, credentials, and certificates. It uses encryption keys from {{site.data.keyword.keymanagementserviceshort}} to encrypt sensitive data and to seal and unseal vaults that hold the secrets. It is preconfigured to send events to the {{site.data.keyword.en_short}} service, allowing customers to set up email or SMS notifications. Moreover, it is automatically configured to forward all API logs to the customer's logging instance. To automate provisioning as code, you can use [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) for [{{site.data.keyword.secrets-manager_short}}](https://registry.terraform.io/modules/terraform-ibm-modules/secrets-manager/ibm/latest){: external}.
```

  ```terraform
  module "secrets_manager" {
    source  = "terraform-ibm-modules/secrets-manager/ibm"
    version = "<version>"

    resource_group_id    = "<resource_group_id>"
    region               = "<region>"
    secrets_manager_name = "<instance_name>"
  }
  ```
  {: codeblock}

3. {{site.data.keyword.sysdigsecure_full_notm}}

>   The {{site.data.keyword.sysdigsecure_full_notm}} instance is pre-configured with Cloud Security Posture Management (CSPM) enabled using the Configuration Aggregator features from the App Configuration instance that is also provisioned as part of this solution. To automate provisioning as code, you can use the [SCC Workload Protection module](https://registry.terraform.io/modules/terraform-ibm-modules/scc-workload-protection/ibm/latest){: external} from [Terraform IBM Modules](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim).

Reviewer 2: same as above

1.  Go the {{site.data.keyword.cloud_notm}} [catalog](/catalog#reference_architecture){: external} and search for the Cloud foundation for security and observability deployable architecture.
1.  Click the tile for the deployable architecture to open the details. The Security & compliance tab lists all of the controls that are included in the deployable architecture.

## Terraform IBM Modules (TIM)
{: #tim-css}

> The services in this architecture are built on pre-built, open-source and enterprise-ready [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim). The full stack source, including all module wiring and input configurations, is available in the [stack-ibm-core-security-services](https://github.com/terraform-ibm-modules/stack-ibm-core-security-services){: external} repository.

Reviewer: ```suggestion
The services in this architecture are based on open-source, enterprise-ready [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim). The complete stack source, including all module wiring and input configurations, is available in the [stack-ibm-core-security-services](https://github.com/terraform-ibm-modules/stack-ibm-core-security-services) repository. {: external}
```

1.  Go the {{site.data.keyword.cloud_notm}} [catalog](/catalog#reference_architecture){: external} and search for the Cloud foundation for security and observability deployable architecture.
1.  Click the tile for the deployable architecture to open the details. The Security & compliance tab lists all of the controls that are included in the deployable architecture.

> ## Terraform IBM Modules

Reviewer 2: ```suggestion
## Terraform IBM Modules (TIM)
```

{: #tim-css}

> The services in this architecture are built on [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim) — open-source, {{site.data.keyword.IBM_notm}}-maintained Terraform modules. The full stack source, including all module wiring and input configurations, is available in the [stack-ibm-core-security-services](https://github.com/terraform-ibm-modules/stack-ibm-core-security-services){: external} repository.

Reviewer 2: ```suggestion
The services in this architecture are built on pre-built, open-source and enterprise-ready [Terraform IBM Modules (TIM)](/docs/ibm-cloud-provider-for-terraform?topic=ibm-cloud-provider-for-terraform-about-tim). The full stack source, including all module wiring and input configurations, is available in the [stack-ibm-core-security-services](https://github.com/terraform-ibm-modules/stack-ibm-core-security-services){: external} repository.
```

also does stack-ibm-core-security-services has a placeholder name like {{site.data.keyword.IBM_notm}} ? If yes then replace it
